HACKING.md

```diff
@@ -32,3 +32,5 @@ You can also set up CLion to do this automatically for you (it is done this corr
 ![CLion setting for ClangFormat](docs/clion_clang_format.png)
 
 ![CLion setting for auto-save formatting](docs/clion_format_on_save.png)
+
+If CLion doesn't format automatically, you can use [clion-format](https://clang.llvm.org/docs/ClangFormat.html) CLI to format it.
```

Comment:
it's clang-format and we have a build target for it:

```
make fix-format
```

which is already described a few lines above https://github.com/omnigres/omnigres/blob/master/HACKING.md#source-code-formatting

Reply:
My bad. That's suffice.